16/umbraco-cms/fundamentals/setup/upgrading/upgrade-details.md

```diff
@@ -2,18 +2,15 @@
 description: Describes how to upgrading existing installations to new versions.
 ---
 
-# Upgrade Instructions
+# Upgrade Details
```

Comment:
This change is not reflected in the Preview. Maybe cz it's **Upgrade Instructions** in the Summary.md file?